chart/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
```

Comment:
Its better to do 'helm create webex-receiver' as starting point

Reply:
will refactor
soon